CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+- Add export option for profile picture
```

Comment:
sorry that i didn't suggest last time, we might want to prefix this with `Desktop:` and also link to the PR `[ #.... ]` ?

Reply:
Will amend the @neolit123 suggestions commit accordingly. I don't think we should have issue numbers in the CHANGELOG as that is end user facing.